src/pages/index.tsx

```diff
@@ -1,86 +1,64 @@
+import Image from "next/image";
 import Link from "next/link";
-import { buttonVariants } from "@/components/ui/Button";
 
 export default function IndexPage() {
   return (
-    <section className="container mx-auto flex max-w-[680px] flex-1 flex-col items-start gap-20 px-4 pt-6 pb-8 md:py-10">
-      <div className="mx-auto flex w-full flex-col items-start gap-2">
+    <section className="items-startpx-4 container mx-auto flex max-w-[680px] flex-1 flex-col pt-6 pb-8 md:py-10">
+      <div className="mx-auto flex w-full flex-col items-start gap-6">
         <h1 className="text-3xl font-extrabold leading-tight tracking-tighter sm:text-3xl md:text-5xl lg:text-6xl">
-          Livestream with LiveKit
+          Hello, broadcaster!
         </h1>
-        <p className="text-lg text-zinc-700 dark:text-zinc-400 sm:text-xl">
-          This is a{" "}
-          <Link
-            href="https://create.t3.gg/"
-            target="_blank"
-            rel="noferrer"
-            className="underline"
-          >
-            T3 Stack
-          </Link>{" "}
-          project built with a collection of technologies including:
-        </p>
-        <ul className="ml-6 list-disc text-lg text-zinc-700 dark:text-zinc-400 sm:text-xl">
-          <li>
+        <div className="flex flex-col gap-6 text-lg text-zinc-700 dark:text-zinc-400 sm:text-xl">
+          <p>
+            This{" "}
             <Link
-              href="https://nextjs.org/"
+              href="https://create.t3.gg/"
```

Comment:
should we write something like "NextJS/T3 app"? Just thinking out loud because I think a lot more people will recognize NextJS than T3.